PCA_Comp_Analysis_Dredging.R

```diff
@@ -190,6 +200,10 @@ clim_to_seasonal<-function(X,season='wet'){
 }
 
 composite<-function(y,x,xcrit,typ='>',outfile=NULL,is.sample=FALSE) {
+  # This functions takes vector x checks if values in X pass a threshold
+  # xcrit and then selects the corresponding values from vector y
+  # to calculate statistics.
+  # vector y the 

```

Comment:
Is the last line of this comment unfinished? Or can be removed?

Reply:
Completed comment.